src/test/groovy/org/prebid/server/functional/model/request/setuid/SetuidRequest.groovy

```diff
@@ -25,4 +30,15 @@ class SetuidRequest {
         request.gdpr = "0"
         request
     }
+
+    @JsonSetter
+    void setGppSid(List<GppSectionId> sectionIds) {
+        StringBuilder stringBuilder = new StringBuilder()
+        for (sectionId in sectionIds) {
+            if (sectionId) {
+                stringBuilder.append(sectionId.value)
+            }
+        }
+        this.gppSid = stringBuilder.toString()
+    }
```

Comment:
I suggest replacing this StringBuilder with join.  Join is faster in this case and improve readability
```
@JsonSetter
void setGppSid(List<GppSectionId> sectionIds) {
    this.gppSid = sectionIds.findAll { it }.collect { it.value }.join()
}
```